.github/workflows/build.yml

```
@@ -0,0 +1,260 @@
+          androidKeyaliasName:  ${{ secrets.ANDROID_STAGING_KEYALIAS_NAME }}
+          androidKeyaliasPass:  ${{ secrets.ANDROID_STAGING_KEYALIAS_PASS }}
+      - name: Rename output APK
+        if: matrix.targetPlatform == 'Android'
+        run: |
+           mkdir -p final/${{ matrix.targetPlatform }} # have to copy renamed APK due to permission issues - can't just rename'
+           cp build/${{ matrix.targetPlatform }}/Android.apk final/${{ matrix.targetPlatform }}/MandoUO-${{ needs.getVersionNumber.outputs.fullVersionNumber }}.apk
+      - name: Upload Android artifact
+        if: matrix.targetPlatform == 'Android'
+        uses: actions/upload-artifact@v4
+        with:
+            name: Build-${{ matrix.targetPlatform }}-MandoUO-${{ needs.getVersionNumber.outputs.fullVersionNumber }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)